packages/terra-open-source-scripts/src/terra-cli/release/releaseMonoRepoHandler.js

```diff
@@ -0,0 +1,37 @@
+const childProcess = require('child_process');
+const path = require('path');
+const fs = require('fs-extra');
+const { promisify } = require('util');
+const Logger = require('@cerner/terra-cli/lib/utils/Logger');
+
+const setupGit = require('./setupGit');
+
+const logger = new Logger({ prefix: '[terra-open-source-scripts:release]' });
+const exec = promisify(childProcess.exec);
+
+// Manipulate lerna publish output to find what packages were released and need to be tagged.
+const getTags = (output) => {
+  const successString = 'Successfully published:\n';
+  const lastIndexOf = output.lastIndexOf(successString);
+  if (lastIndexOf < 0) {
+    return null;
+  }
+  const substring = output.substring(lastIndexOf + successString.length);
+  const withoutDashes = substring.replace(/ - /g, '').trim();
+  return withoutDashes.split('\n');
+};
+
+module.exports = async () => {
+  await fs.writeFile(path.join(process.env.HOME, '.npmrc'), `//registry.npmjs.org/:_authToken=${process.env.NPM_TOKEN}`, 'utf-8');
```

Comment:
consider commenting this file up to describe what you're doing and why for any future travelers.